appIcons.js

```diff
@@ -753,7 +750,11 @@ var taskbarAppIcon = new Lang.Class({
             else
                 buttonAction = this._dtpSettings.get_string('click-action');
         }
-        
+
+        let appCount = this.getAppIconInterestingWindows().length;
```

Comment:
Please modify the `appIsRunning` assignation to use the new `appCount` variable, so we don't call `this.getAppIconInterestingWindows()` twice in a row

Reply:
I did it, thank you.